action.yml

```diff
@@ -5,23 +5,39 @@ inputs:
     description: "The runner version (The actual version is controlled via project/build.properties)"
     required: true
     default: 1.10.6
+outputs:
+  sbt-toolpath:
+    description: "Where sbt tools live"
+    value: ${{ steps.cache-paths.outputs.sbt_toolpath }}
+  cache-hit:
+    description: "If sbt was found in the tool cache or the action/cache"
+    value: ${{ steps.cache-tool-dir.outputs.cache-hit == 'true' || steps.cache-dir.outputs.cache-hit == 'true' }}
+  cache-key:
+    description: "The cache-key for the sbt version (unless sbt was found in the tool cache)"
+    value: ${{ steps.cache-tool-dir.outputs.cache-hit != 'true' && steps.cache-paths.outputs.sbt_cachekey || '' }}
```

Comment:
What's the point of these? Unless it's necessary, I want to avoid exposing anything that users would try to use and we would need to maintain going forward.

Reply:
actions/cache exposes a hit/key, you're acting as a cache, it's odd that you aren't exposing your cache-hit and cache-key values.

Similarly, it seems reasonable to let people know where you put things instead of making them search through `$GITHUB_PATH` (or `$PATH`)

Reply:
In case of action/cache, the build users are using it programmatically to cache arbitrary things whereas setup-sbt exists merely to emulate the previous base image wherein `sbt` runner was on the `PATH` somehow. I want the action to be as simple as possible so the only thing build user can expect is that, and the only way user can interact with the action is via `sbt`-on-`PATH`.

Reply:
Fair. I'll drop.